lego/apps/users/views/student_confirmation.py

```diff
@@ -90,18 +90,22 @@ def create(self, request, *args, **kwargs):
         user.student_username = student_confirmation['student_username']
         course = student_confirmation['course'].lower()
 
+        has_grade_group = user.has_grade_group
+
         if course == constants.DATA:
             course_group = AbakusGroup.objects.get(name=constants.DATA_LONG)
             course_group.add_user(user)
 
-            grade_group = AbakusGroup.objects.get(name=constants.FIRST_GRADE_DATA)
-            grade_group.add_user(user)
+            if not has_grade_group:
+                grade_group = AbakusGroup.objects.get(name=constants.FIRST_GRADE_DATA)
+                grade_group.add_user(user)
         else:
             course_group = AbakusGroup.objects.get(name=constants.KOMTEK_LONG)
```

Comment:
You can remove this line too! (and the same for `DATA_LONG`), since the users isn't supposed to be members of `Datateknologi` or `Kommunikasjonsteknologi` (becuse of the tree structure, uses will get the proper permissions from being in a grade group).

It is also not supported by the bump script: https://github.com/webkom/lego/pull/1249